distribution/scenes/textures/patterns/slope.pov

```diff
@@ -112,22 +112,29 @@ texture {
 // in the texture_map of the height_field below.
 // The altitude values have been switched around because we
 // want the snow near the top and not near the bottom.
-#declare Rock2Snow_Texture =
-texture {
-   slope {
-      -y*2, 0, 0.5
-      altitude y, 1.000, 0.400
-   }
-   texture_map {
-      [0.75, Snow_Texture]
-      [0.75, Rock_Texture]
-   }
+#declare Rock2Snow_Texture = texture {
+    slope {
+        -y*2, 0, 0.5
+        altitude y, 1.000, 0.400
+    }
+    texture_map {
+        [0.75, Snow_Texture]
+        [0.75, Rock_Texture]
+    }
 }
 
 
-// Here is the mountain.
+// Here is the mountain. height_field passes x & y,
+// (u,v unit square) coordinates to Fn01 function.
+#include "functions.inc"
+#declare Fn00 = function (x,y,z) {
+    pow(max(0,1-(x*x+y*y+(f_wrinkles(x,y,z)*2.5))),2.5)
+}
+#declare Fn01 = function (x,y,z) {
+    Fn00((x-0.5)*2,(y-0.5)*2,z)
+}
 height_field {
-   "slopemt_dat_36.jpg"
+   function 1200, 1200 { Fn01(x,y,z) }
```

Comment:
Not sure if we want to replace the image-based height field with a function-based height field. Makes rendering of this particular scene easier, but probably isn't the standard use case for height fields.